upgrade-agent/upgradeproto/upgrade.proto

```diff
@@ -11,8 +11,8 @@ service Update {
 }
 
 message ExecuteUpdateRequest {
-  string kubeadm_url = 1;
-  string kubeadm_hash = 2;
+  reserved 1;
+  reserved 2;
```

Comment:
Nit: field names should also be reserved, https://protobuf.dev/programming-guides/proto3/#fieldreserved.
```suggestion
  reserved 1, 2;
  reserved "kubeadm_url", "kubeadm_hash";
```